doc/source/configuration/wazuh.rst

```diff
@@ -291,7 +291,7 @@ Verification
 ==============
 
 The Wazuh portal should be accessible on port 443 of the Wazuh
-manager’s IPs (using HTTPS, with the root CA cert in ``etc/kayobe/ansible/wazuh/certificates/root-ca.pem``).
+manager’s IPs (using HTTPS, with the root CA cert in ``etc/kayobe/ansible/wazuh/certificates/wazuh-certificates/root-ca.pem``).
```

Comment:
Is this path correct?

Reply:
Yes. `wazuh-certs-tool.sh` is creating this directory.

Reply:
What path would it be if using custom certs?

Reply:
`"{{ playbook_dir }}/wazuh/custom_certificates"` for example: `etc/kayobe/ansible/wazuh/custom_certificates/root-ca.pem`

Reply:
Is it not also present at `etc/kayobe/ansible/wazuh/certificates/root-ca.pem` when generating certs? I'm looking for a standard path to reference in both cases.

Reply:
When certs are autogenerated they are generated to `etc/kayobe/ansible/wazuh/certificates/wazuh-certificates/`.  Path `etc/kayobe/ansible/wazuh/certificates` is used by Wazuh playbooks to download wazuh-certs-tool.sh, and this tool is creating wazuh-certificates directory and generates certs there, later certs are copied to wazuh host. 
In custom cert scenario, certs are stored in `etc/kayobe/ansible/wazuh/custom_certificates` and later are copied to wazuh host.

You can have `etc/kayobe/ansible/wazuh/certificates/root-ca.pem` and use it for signing certs generated by `wazuh-certs-tool.sh` (tool itself is supporting it), but this is manual action, as upstream roles currently don't support this from ansible perspective. 

So tl;dr answer is: no.

As for standard path. I think it is handy to have this,  but right now we are in place where cert tool is started on every playbook run, so it strong candidate for follow up task.